bundles/binding/org.openhab.binding.fritzboxtr064/src/main/java/org/openhab/binding/fritzboxtr064/internal/FritzboxTr064Binding.java

```diff
@@ -138,7 +138,13 @@ public void activate(final BundleContext bundleContext, final Map<String, Object
         this._pw = fboxpw;
         this._user = fboxuser;
         this._url = fboxurl;
-        this._pbid = Integer.valueOf(fboxphonebookid);
+        
+        try{
+            this._pbid = Integer.valueOf(fboxphonebookid);
+    	}catch(NumberFormatException ex){ //set fallback to 0
```

Comment:
Could you apply the auto-formatter to insert spaces around the curly brackets so that this matches the style of the rest of the code?